appveyor.yml

```diff
@@ -6,19 +6,32 @@ cache:
 - '%USERPROFILE%\.nuget\packages -> ReactWindows\**\project.json'
 - ChakraCore\Build\VcBuild -> ChakraCore\lib\Common\*.h
 
+install:
+  - npm install -g npm@latest
+  - npm i
+  - ps: curl -o C:\winium.zip https://github.com/2gis/Winium.Desktop/releases/download/v1.6.0/Winium.Desktop.Driver.zip
+  - ps: mkdir C:\winium
+  - ps: Add-Type -A 'System.IO.Compression.FileSystem'
+  - ps: '[IO.Compression.ZipFile]::ExtractToDirectory("C:\winium.zip", "C:\winium")'
+
 clone_script:
 - ps: git clone -q $("--branch=" + $Env:APPVEYOR_REPO_BRANCH) $("https://github.com/" + $Env:APPVEYOR_REPO_NAME + ".git") $Env:APPVEYOR_BUILD_FOLDER
 - ps: if (!$Env:APPVEYOR_PULL_REQUEST_NUMBER) {$("git checkout -qf " + $Env:APPVEYOR_REPO_COMMIT)}
 - ps: if ($Env:APPVEYOR_PULL_REQUEST_NUMBER) {git fetch -q origin +refs/pull/$($Env:APPVEYOR_PULL_REQUEST_NUMBER)/merge; git checkout -qf FETCH_HEAD}
 - ps: git submodule update -q --init --recursive
 
 before_build:
+- ps: $env:playgroundNet46_dir=$Env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46"
+- ps: $env:bundle_dir=$Env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46\ReactAssets"
 - ps: >-
     Get-ChildItem .\ChakraCore\Build\VcBuild * -recurse | ForEach-Object{$_.LastWriteTime = get-date}
 
     nuget restore ReactWindows\ReactNative.sln
+- npm i -g react-native-cli
 
 build_script:
+- ps: mkdir $env:bundle_dir
```

Comment:
build the releasebundle for Playground.Net46 copying it to the Playground.Net46 directory...